client/src/disabledGroups.ts

```diff
@@ -4,15 +4,17 @@ import { DateTime } from "luxon";
 export interface PotentiallyDisabledGroup {
   isUpcoming: boolean;
   isExpired: boolean;
+  isJoinable: boolean;
   group: Group;
 }
 
 function toPotentiallyDisabledGroup(group: Group): PotentiallyDisabledGroup {
   const now = DateTime.local();
   const isExpired = (group.disabledAfter && DateTime.fromISO(group.disabledAfter) <= now) || false;
+  const isJoinable = (group.joinableFrom) ? DateTime.fromISO(group.joinableFrom) <= now && !isExpired : !isExpired;
```

Comment:
Not sure, but wouldn't this mean that upcoming cards are always enabled when joinableFrom is not set?

Reply:
had fixed that already but the push was missing... should be correct now.